libbeat/processors/actions/decode_json_fields.go

```diff
@@ -17,18 +17,21 @@ type decodeJSONFields struct {
 	fields       []string
 	maxDepth     int
 	processArray bool
+	target       string
```

Comment:
What if I want to target the root of the event? How would I configure this?

I _think_ if you make it a `*string` then you can determine the difference between `target` being set but empty and `target` not present at all.